src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -921,6 +981,22 @@ static int rbd_nbd(int argc, const char *argv[])
     return EXIT_FAILURE;
   }
 
+  return 0;
+}
+
+static int rbd_nbd(int argc, const char *argv[])
+{
+  int r;
+
+  vector<const char*> args;
+
+  argv_to_vec(argc, argv, args);
+
+  r = parse_args(args);
+  if (r == EXIT_FAILURE) {
```

Comment:
`parse_args()` should also return a negative error code instead of `EXIT_FAILURE`.